src/editors/containers/ProblemEditor/data/apiHooks.ts

```diff
@@ -0,0 +1,19 @@
+import { useMutation } from '@tanstack/react-query';
+import { getConfig } from '@edx/frontend-platform';
+import api from '@src/editors/data/services/cms/api';
+
+const getApiBaseUrl = () => getConfig().STUDIO_BASE_URL;
+
+export const useValidateInputBlock = () => useMutation({
+  mutationFn: async (title) => {
+    try {
+      const res = await api.validateBlockNumericInput({ studioEndpointUrl: `${getApiBaseUrl()}`, data: { formula: title } });
+      return res.data;
```

Comment:
```suggestion
      return camelCaseObject(res.data);
```
Convention in this repo is to use camelCase on the API responses, so that `is_valid` becomes `isValid`.

Reply:
thank you, I made the change